src/main/scala/soc/playerRepository/PlayerContext.scala

```diff
@@ -78,6 +99,82 @@ class PlayerContext[GAME <: Inventory[GAME], PLAYER <: Inventory[PLAYER]](val na
     }
   }
 
+  def getEvent(position: Int, moveResult: MoveResult): GameEvent = {
```

Comment:
This should be integrated into `GameBehavior`. Because all players receive the same game updates, the `GameEvent` should be bundled in `GameBehavior` and then streamed to all observers.